docs/en/operation/Journal.md

```diff
@@ -343,6 +340,10 @@ Therefore, Alluxio primary master will not create checkpoints by default.
 Restarting the current primary master to transfer the leadership to another running master periodically
 can help avoiding primary master journal logs from growing unbounded when Alluxio is running in HA mode.
 
+Starting from Alluxio 2.4.0, Alluxio primary master can download checkpoints from standby masters
+and install in its journal folder in embedded journal HA mode. By doing so, Alluxio primary master 
+can truncate its journal size without causing temporary service unavailability.
```

Comment:
Could you clarify if the user needs to do anything on their end or if this is an automatic optimization?

Reply:
Modify the words, please take another look, thanks!